### ISSUE TITLE
Auth0 Lock "Modal" is not showing "Signup" tab when opened from second time onward

### ISSUE THREAD

**Issue by bjspatel** [auth0/lock#1870]
### Description
We use the auth0-lock in our UI, and open the modal for the first time it shows all the fields as expected along with the 3 social login buttons.
Next, when we close the modal by pressing the Esc key and then open the modal again I don't see the "Login | Signup" tabs on top of the modal, and also I don't see the first Social login button for Twitter as well.

### Reproduction

> - Code sample to reproduce the issue
```
    const options = {
      closable: true,
      allowLogin: true,
      rememberLastLogin: true,
      auth: {
        redirect: true,
        responseType: "code",
        redirectUrl: urls.auth0Redirect,
      },
      theme: {
        logo: "<our_logo_url>",
        primaryColor: "#0099FF",
      },
      allowedConnections: [
        "twitter",
        "google-oauth2",
        "github",
        "Username-Password-Authentication",
      ]
    };
    const auth0Lock = new Auth0Lock(config.auth0ClientID, urls.auth0Domain, options);

```
> Detail the steps taken to reproduce this error, what was expected, and whether this issue can be reproduced consistently or if it is intermittent.

**Steps**:
- Open the auth0 modal for the first time
- Press `Esc` key to close the auth0 modal
- Open the auth0 modal for the second time

**Current Behavior**
- For the first time, it shows all UI controls correctly
- For the second time, it doesn't show "Login|Signup" tabs, and also it doesn't show the first Social 
 Login button for "Twitter"

**Expected Behavior**
- Auth0 modal should be consistent in all the events of opening it

**Screenshots**

- First Attempt
![first_attempt](https://user-images.githubusercontent.com/1701475/82525831-0c0ba480-9b50-11ea-8770-1cabc80ec0f5.png)

- Second Attempt
![second_attempt](https://user-images.githubusercontent.com/1701475/82525856-1cbc1a80-9b50-11ea-8ebf-a624a456ee96.png)

### Environment

- **Version of this library used:** 11.24.0
- **Version of the platform or framework used, if applicable:** React 16.12.0
- **Other relevant versions (language, server software, OS, browser):** Browser Chrome 81.0.4044.138
- **Other modules/plugins/libraries that might be involved:** -

Looks like this is a bug on the auth0 side. Please suggest if this can be fixed by doing something different from our side. Thanks!

**Comment by adamjmcgrath** [comment]
Hi @bjspatel - thanks for raising this

I'm not able to reproduce the issue with those options and those social connections configured on my tenant.

Do you have a running example somewhere I can look at?

**Comment by bjspatel** [comment]
Hi @adamjmcgrath 
Our application is running at https://boards-dev.weavr.ai/brijesh/board/cloud-computing-pile
Please try clicking on different operations like "Sign in", or "+" icon to open the modal and observe the behavior.

**Comment by stevehobbsdev** [comment]
@bjspatel We had an issue with the header height calculation in 11.24.0, which we think we've fixed in 11.24.1 - could you try updating and let us know the results?

**Comment by bjspatel** [comment]
@stevehobbsdev 
I deployed the auth0-lock upgrade. It's now running 11.24.1 version now.
The issue is still present. I also tried this on firefox. There also it's showing this behavior from third attempt to open the modal.
You can try on the above link. Also please let me know if you need any other information.
Thanks!
